api/yarn.lock

```diff
@@ -0,0 +1,5310 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Looks like some of us are using `yarn` and some `npm`. I started off with `npm`, heard it's better than `yarn` nowadays. Thoughts?

Reply:
✋  guilty. `npm` is old school, when `yarn` came along, it was new and shiny so I still think of it that way. 
Both work about the same.. so I don't mind which we use. I'll stop yarning.